lib/linux.js

```diff
@@ -38,7 +38,17 @@ const trash = async (filePath, trashPaths) => {
 };
 
 export default async function linux(paths) {
-	const mountPointMap = new Map(procfs.processMountinfo().map(info => [info.devId, info.mountPoint]));
+	// • filter irrelevant
+	// • “first one wins” if id comes along twice
+	const mountPointMap = new Map();
+
+	for (const m of Object.values(procfs.processMountinfo())
+		.filter(m => !/^\/(snap|run|sys|proc|dev)($|\/)/.test(m.mountPoint))) {
```

Comment:
Nitpick: Would be better to just add an if-statement in the loop for this and use `continue`.

Reply:
Well, some people love chaining, some love if's... (usually I am in the if-camp, too)
Anyway, yes, changed to that.

And added `/var/snap` (as another and problem causing real-world encounter.)